doc/development/release_distribution.rst

```diff
@@ -46,6 +46,17 @@ For every change that is going to be part of your release, make sure that:
 
 Please make a normal PR to ``master`` correcting anything missing from these points and have it merged before you begin the release, if necessary.
 
+.. _update-requirement:
+
+Updating the Requirements
+-------------------------
+
+Ensure that QuTip's tests pass on the oldest version supported in the requirements.
+On major and minor version, requirements can be updated to remove support of older version of some packages.
+We follow `NEP29`_ for minimum supported versions.
+
+.. _NEP29: https://numpy.org/neps/nep-0029-deprecation_policy.html
```

Comment:
I would also suggest to be more specific on what we mean by "we follow NEP29". Referencing NEP29 is great but I think it is best to be explicit and explain what we support and how. For example, adding something similar to this may help:

> This project supports:
> 
> - All minor versions of Python released 42 months prior to the project, and at minimum the two latest minor versions.
> - All minor versions of numpy released in the 24 months prior to the project, and at minimum the last three minor versions.
> 
>     In setup.py, the python_requires variable should be set to the minimum supported version of Python. All supported minor versions of Python should be in the test matrix and have binary artifacts built for the release.
> 
>     Minimum Python and NumPy version support should be adjusted upward on every major and minor release, but never on a patch release.
> 

This is copied from the NEP29.